dashboard/app/controllers/activities_controller.rb

```diff
@@ -49,6 +49,13 @@ def milestone
       return
     end
 
+    # If a student in the pilot is submitting work on an AI-enabled level, trigger the AI evaluation job.
+    is_ai_experiment_enabled = current_user && Experiment.enabled?(user: current_user, experiment_name: 'ai-rubric')
+    is_level_ai_enabled = EvaluateRubricJob.ai_enabled?(@script_level)
+    if is_ai_experiment_enabled && is_level_ai_enabled && params[:submitted] == 'true'
```

Comment:
Should this check that the user has enabled AI? I guess being in the experiment is a good enough approximation of that for now

Reply:
Good question! asking here: https://codedotorg.slack.com/archives/C050HE4QDHR/p1696881716154169

Reply:
I think so. It's just a matter of `!teacher.ai_rubrics_disabled` for each teacher, I think. Can this logic be part of the section model? I'm not sure how to make that work in the milestone context.